Automattic/wp-calypso
# Fix jetpack masterbar animations and missing icons

## Proposed Changes

* Add missing icons for Growth bundle in Jetpack header
* Fix animation for sections on header

## Why are these changes being made?

* The current header does not match the jetpack.com one and the animations are not working right

## Testing Instructions

1. Use the Calypso live link to go to `/pricing`
2. Open up the header and ensure the animation of the items loading in looks good and that the Growth bundle has all it's icons
![image](https://github.com/user-attachments/assets/f5278cd1-6d7d-49ab-8b34-bbc2812afd2d)


## Pre-merge Checklist
